### PR TITLE
Ignore copying README & LICENSE files if they already exist

### LINKED ISSUE #36: Ignore copying README & LICENSE files if they already exist
If SwiftPlate is run in a repo that has been cloned from an empty GitHub repo, it will already contain a `README.md` and `LICENSE` files, which will cause SwiftPlate to error. What should happen is that if any (or both) of those files already exists, SwiftPlate should **not** attempt to overwrite them.

### PR DESCRIPTION
**Pull request by alexaubry**
If a README.md and/or a LICENSE files already exist in the destination directory, SwiftPlate will no longer attempt to overwrite them.

This fixes #36.

### CODE REVIEW

**Comment by kiliankoe** [comment]
GitHub does not require the capitalization of these files. Would it therefore make sense to check the lowercased names against each other instead?

**Comment by alexaubry** [comment]
Good point! I made the check case-insensitive.

File: main.swift [hunk 1]
```diff
@@ -361,9 +361,22 @@ do {
     }
     
     try performCommand(description: "Copying template folder") {
+        let ignorableItems = ["readme.md", "license"]
```
Comment (JohnSundell) [inline main.swift]: How about using a `Set` here instead, to improve lookup time for `contains()`. Minor detail I guess 🙂

File: main.swift [hunk 2]
```diff
@@ -361,9 +361,22 @@ do {
...
+        let ignoredItems = try fileManager.contentsOfDirectory(atPath: destination).map {
```
Comment (JohnSundell) [inline main.swift]: Why is this needed? Can't we just compare against `ignorableItems` directly?

File: main.swift (hunk 1 above)
Comment (alexaubry) [inline main.swift]: Great remark. I hadn't thought about it 🙃

File: main.swift (hunk 2 above)
Comment (alexaubry) [inline main.swift]: This builds the list of the files that won't be copied by checking which ignorable items are contained in the destination folder.

I thought it was better (and faster) to build the list once and compare every `itemName` against it rather than checking if `itemName` is ignorable and then calling `fileExists(atPath:)`.

Comment (alexaubry) [inline main.swift]: My initial implementation was:

~~~swift
if ignorableItems.contains(itemName) {
    guard fileManager.fileExists(atPath: destinationPath) == false else {
        continue
    }
}
~~~

But it has 2 problems IMHO:

- not the best syntax (more indents)
- case sensitive (`Readme.md` won't be matched for instance)
 
Let me know if you think this implementation's better 🙂
